spec/models/user_spec.rb

```diff
@@ -1,7 +1,12 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  let(:user) { Fabricate (:user) }
+  let(:user) { User.new(username: "han",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.